blog/keycloak-newsletter-2.ftl

```diff
@@ -0,0 +1,81 @@
+<#--
+title = Keycloak Community Newsletter #2
+date = 2019-05-02
+publish = false
+author = Sébastien Blanc
+-->
+
+<p>
+A lot of news to share in this second edition, so fasten your seatbelt and let's go ! 
+</p>
+
+
+<h2>News from the community</h2>
+<p>
+First of all, we would like to thank the whole community that has contributed to this edition by sharing their links, tips, etc ...
+</p>
+<p>
+More and more API Management/Gateway solutions can now be integrated with Keycloak. Recently, <a href="https://www.getambassador.io/">Ambassador</a>, a Open Source Kubernetes-Native API Gateway built on the Envoy Proxy, has published a <a href="https://www.getambassador.io/tutorials/github-sso/">quickstart</a> that shows the Keycloak's integration.   
+</p>
+<p>
+Good news for Python users, Akhil Lawrence has created a <a href="https://github.com/akhilputhiry/keycloak-client/releases">Python Keycloak Adapter Client</a>. Be sure to check out the really nice <a href="https://keycloak-client.readthedocs.io/en/latest/">documentation</a> that comes with a lot of usage examples.  
+</p>
+<p>
+Like any other project, setting up clustering can be somehow complex. Liqiang has shared with us his <a href="https://github.com/fit2anything/keycloak-cluster-setup-and-configuration">setup and configuration tips</a> , thanks again for sharing your knwoledge with the community ! 
+</p>
+<P>
+The existing <a href="https://github.com/helm/charts/tree/master/stable/keycloak">Keycloak Helm Chart</a> has now been deprecated and is now replaced by the <a href="https://github.com/codecentric/helm-charts">one</a> managed by CodeCentric.
+</p>
+<p>
+Dmitry Telegin has created a really nice example on how you can dynamically brand your login them for Keycloak, check out the repository <a href="https://github.com/dteleguin/keycloak-dynamic-branding">here</a>.
+</p>
+<p>
+Hayri Cicek has written a nice introductory <a href="https://kodnito.com/posts/microprofile-jwt-with-keycloak/">article</a> on how to use Keycloak and MicroProfile. It even show the usage of the brand new Client Scope "microprofile-jwt" which has been added in Keycloak 6.0.0.
+</p>
+
+
+<h2>News from the project</h2>
+<p>
+Keycloak 6.0.1 has been <a href="https://www.keycloak.org/2019/04/keycloak-601-released.html">released</a>. We know that some people were a bit confused by our new versioning schema, hopefully, this <a href="https://www.keycloak.org/2019/04/versioning.html">blog post</a> will clear it up. 
+</p>
+<p>
+We have some exciting news to share : we started the process to integrate the <a href="https://www.cncf.io/">Cloud Native Computing Foundation</a>. This will boost our community adoption with the ultimate goal of becoming the de facto OAuth2/OpenID Connec solutin within the Open Source and Cloud Native ecosystem. But we will also be more involved in the differents Security SIGs and benefit from the CNCF CI infrastructure, i.e the <a href="github.com/cncf/cluster">CNCF Cluster</a>. 
```

Comment:
Did you ask B about this?

Change to:
We started the proposal process for Keycloak to be accepted into the Cloud Native Computing Foundation (CNCF). This will hopefully significantly boost our community adoption with our ultimate goal of becoming the de facto solution for OAuth2/OpenID Connect within Open Source and Cloud Native.

Reply:
B is okay with this ;) and let me add your change.

Reply:
Do you want me to replace the whole paragraph with your suggestion ?

Reply:
I'd say so. I wouldn't mention Security SIG or CNCF CI infrastructure at this point.